core/arch/arm/kernel/generic_boot.c

```diff
@@ -72,9 +72,8 @@
 
 #define PADDR_INVALID		ULONG_MAX
 
-#ifdef CFG_BOOT_SECONDARY_REQUEST
-uint32_t ns_entry_addrs[CFG_TEE_CORE_NB_CORE] __data;
-#endif
+static paddr_t ns_entry_addrs[CFG_TEE_CORE_NB_CORE] __early_bss;
```

Comment:
preserve `#if defined(CFG_BOOT_SECONDARY_REQUEST)`.
Otherwise these variables are uselessly linked in the early bss section.